floss/stackstrings.py

```diff
@@ -57,10 +64,33 @@ def apicall(self, emu, op, pc, api, argv):
         stack_buf = emu.readMemory(stack_top, stack_size)
         self.ctxs.append(CallContext(op.va, stack_top, stack_bottom, stack_buf))
 
-
-def extract_call_contexts(vw, fva):
+    def posthook(self, emu, op, endpc):
+        self.get_context_via_mov_heuristic(emu, op, endpc)
+
+    def get_context_via_mov_heuristic(self, emu, op, endpc):
```

Comment:
this doesn't return a context, so we should probably name it differently (reserve `get_*` for fetching some value). maybe `update_stack_heuristics()`.